website/docs/r/encryption_at_rest.html.markdown

```diff
@@ -59,13 +59,56 @@ resource "mongodbatlas_encryption_at_rest" "test" {
 
 **NOTE**  if using the two resources path for cloud provider access, `cloud_provider_access_setup` and `cloud_provider_access_authorization`, you may need to define a `depends_on` statement for these two resources, because terraform is not able to infer the dependency.
 
-```hcl
+```terraform
```

Comment:
That's correct, would be good to change hcl to terraform since hcl is more generic. Here the [reference](https://github.com/hashicorp/vscode-terraform/pull/225#issuecomment-650265457)

Reply:
Added an item to backlog for later - INTMDB-257